src/subgraph/dex/actions/getTokenInfo.ts

```diff
@@ -0,0 +1,36 @@
+import { ApolloClient } from "@apollo/client/core";
+import { Maybe, TokenInfo } from "../../../types";
+import * as queries from "../queries";
+import { TokenInfoDto } from "../types";
+import { performQuery } from "../../helpers";
+import { getLogger } from "../../../utilities";
+
+const logger = getLogger().withTag("subgraph:dexClient:getTokenInfo");
+
+export const getTokenInfo = async <T>(
```

Comment:
nit: use a more descriptive name than <T> for the generic

Reply:
This is the `TCacheShape` used by the `ApolloClient` so I modified it to be the full name not just `T` for clarity